<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchTab</class>
 <widget class="QWidget" name="SearchTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>650</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="searchCriteriaGroup">
     <property name="title">
      <string>Search Criteria</string>
     </property>
     <layout class="QVBoxLayout" name="searchCriteriaLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="topRow">
        <item>
         <widget class="QLabel" name="searchLabel">
          <property name="text">
           <string>Search:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="search_edit">
          <property name="placeholderText">
           <string>Enter search term...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="search_btn">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clear_btn">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="checkRow">
        <item>
         <widget class="QLabel" name="searchInLabel">
          <property name="text">
           <string>Search in:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="search_customer_check">
          <property name="text">
           <string>Customer</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="search_job_check">
          <property name="text">
           <string>Job #</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="search_po_check">
          <property name="text">
           <string>PO #</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="search_desc_check">
          <property name="text">
           <string>Description</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="search_drawing_check">
          <property name="text">
           <string>Drawings</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="mode_row_widget" native="true">
        <layout class="QHBoxLayout" name="modeRowLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="modeLabel">
           <property name="text">
            <string>Mode:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="search_all_radio">
           <property name="text">
            <string>Search All Folders</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="search_strict_radio">
           <property name="text">
            <string>Strict Format (faster)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="legacy_options_widget" native="true">
        <layout class="QHBoxLayout" name="legacyOptionsLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="alsoSearchLabel">
           <property name="text">
            <string>Also search:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="search_blueprints_check">
           <property name="text">
            <string>Blueprints directories</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="searchResultsGroup">
     <property name="title">
      <string>Search Results</string>
     </property>
     <layout class="QVBoxLayout" name="searchResultsLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QTableWidget" name="search_table">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Customer</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Job #</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>PO #</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Description</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Drawings</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="search_status_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="search_progress">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
